aries-site/src/examples/templates/persistent-navigation/HeaderNavExample.js

```diff
@@ -0,0 +1,269 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Avatar,
+  Box,
+  Button,
+  Grid,
+  Header,
+  Heading,
+  List,
+  Menu,
+  Nav,
+  ResponsiveContext,
+  Stack,
+  Text,
+} from 'grommet';
+import {
```

Comment:
icons are not being used so maybe lets save some lines and take these imports out. as well as the icon.s in pages ?

Reply:
Thanks. Will fix.

Reply:
@britt6612 This should be resolved